android-server-app/app/src/test/java/io/appium/espressoserver/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package io.appium.espressoserver;
```

Comment:
This is generated by default. I think we should leave it there in the event that we want to write  unit tests and this is there as a stub.